CHANGES.md

```diff
@@ -1,7 +1,65 @@
 # Changelog
 
+# 1.2.4 (core) / 0.18.4 (libraries)
+
+### New
+
+- Further performance improvements to asset reconciliation sensor.
+- Performance improvements to asset backfills with large numbers of partitions.
+- `AssetsDefinition.to_source_assets` to method convert a set of assets to `SourceAsset` object.
+- (experimental) A partition mapping that defines dependency relationships between different `MultiPartitionsDefinitions` now exists.
+- [dagster-mlflow] Removed the `mlflow` pin from the `dagster-mlflow` package.
+- [ui] Syntax highlighting now supported in rendered markdown code blocks (from metadata)
+
+### Bugfixes
+
+- When using the `build_asset_reconciliation_sensor`, in some cases duplicate runs could be produced for the same partition of an asset. This has been fixed.
+- When using Pythonic configuration for resources, aliased field names would cause an error. This has been fixed.
+- Fixed an issue where context.partition_time_window
+- [dagster-dbt] In some cases, use of ephemeral dbt models could cause the dagster representation of the dbt dependency graph to become incorrect. This has been fixed.
+- [celery-k8s] Fixed a bug that caused JSON deserialization errors when an Op or Asset emitted JSON that doesn't represent a `DagsterEvent`.
+- Fixed an issue where launching a large backfill while running `dagster dev` would sometimes fail with a connection error after running for a few minutes.
+- Fixed an issue where `dagster dev` would sometimes hang when running Dagster code that attempted to read in input via stdin.
+- Fixed an issue where runs that take a long time to import code would sometimes continue running even after they were stopped by [run monitoring](https://docs.dagster.io/deployment/run-monitoring#run-monitoring) for taking too long to start.
+- Fixed an issue where `AssetSelection.groups()` would simultaneously select both source and regular assets and consequently raise an error.
+- Fixed Pythonic config objects erroring when omitting optional values rather than specifying `None`.
```

Comment:
```suggestion
- Fixed an issue where `BindResourcesToJobs` would raise errors encapsulating jobs which had config specified at definition-time.
- Fixed Pythonic config objects erroring when omitting optional values rather than specifying `None`.
```

Added config entry for cherry-picked fix.

Reply:
Not doing "Commit suggestion" but including this in commit I pushed up